lib/ansible/modules/windows/win_domain.ps1

```diff
@@ -75,6 +75,17 @@ If(-not $forest) {
         $iaf = Install-ADDSForest @install_forest_args
 
         $result.reboot_required = $iaf.RebootRequired
+
+        # The Netlogon service is set to auto start but is not started. This is
+        # required for Ansible to connect back to the host and reboot in a
+        # later task. Even if this fails Ansible can still connect but only
+        # with ansible_winrm_transport=basic so we just display a warning if
+        # this fails.
+        try {
+            Start-Service -Name Netlogon
```

Comment:
I wonder if its worth 'treading lightly' and only calling 'Start-Service' if Netlogon isn't currently running?  Would potentially avoid an unneeded warning if using Basic auth?

Reply:
Probably not, if the service is already running it shouldn't be doing anything. If an error comes up then it's probably a sign of a bigger issue and we should know about it.